frontend/src/views/Results.tsx

```diff
@@ -273,7 +273,9 @@ function GameResultsPage() {
           <ResizableMonacoEditor
             onLanguageChange={null}
             onCodeChange={null}
-            codeMap={null}
+            getCurrentLanguage={null}
+            defaultCodeMap={null}
+            currentProblem={0}
```

Comment:
so for now the results page only shows the code for the first problem? if you changed this to be 1 would that cause it to show the code for the second problem (or n/a if it doesn't exist)?

Reply:
Actually, the defaultCode will be displayed regardless of what currentProblem is set to, as the editor defaults to the defaultCode and only uses the codeMap and currentProblem after one of the callbacks/effects are triggered, which shouldn't happen on the Results page. I just put a 0 there because ints can't be null. Also to answer your question in the main comment, I just merged 146 into stagingNumProblems.